tests/unit/ai/review/test_context_integration.py

```
@@ -0,0 +1,159 @@
+        return self.path / child
+
+
+def _run_git(
+    git_repo: GitRepo,
+    *args: str,
+    **kwargs: Any,
+) -> subprocess.CompletedProcess[bytes]:
+    """Run git in the test repo with a single security suppression."""
+    kwargs.setdefault("cwd", git_repo.path)
+    kwargs.setdefault("check", False)
+    return subprocess.run(  # nosec B603 - argv is assembled from test fixture paths only; shell=False; exercises real git in a temp repo, not user input
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)